RELEASE-NOTES.txt

```diff
@@ -1,3 +1,4 @@
 * App Settings Privacy: Added explanation for the 'Remove Location from Media' feature.
 * History / Revisions: Added swipe action to the revisions browser
-* Added a notice and undo button when loading a revision from a post's history.
\ No newline at end of file
+* Added a notice and undo button when loading a revision from a post's history.
+* Added the possibility to store and check if a reader site is a feed.
```

Comment:
Maybe we don't need the part about storing now?

Reply:
Yep! I fixed it!